### ISSUE TITLE
BHL export lacks author id

### ISSUE THREAD

**Issue by rdmpage** [rdmpage/biostor-lite#3]
Bug reported by @mlichtenberg

```
Rod,

The Biostor API method that returns a list of articles for an item is not including an “id” attribute for each author.

Example Request: http://biostor.org/itemarticles.php?item=311410
Example Response:

{
"ItemID": "311410",
"articles": [
{
"reference_id": "274637",
"title": "Wurmbea fluviatilis (Colchicaceae), a new riverine species from the Gascoyne region of Western Australia",
"genre": "article",
"year": 2011,
"date": 2011,
"authors": [
{
"lastname": "Macfarlane",   ß NO “id” ATTRIBUTE HERE!!
"forename": "T D"
},
{
"lastname": "Case",   ß NO “id” ATTRIBUTE HERE!!
"forename": "A L"
}
],
"secondary-title": "Nuytsia",
.
.
.

Looking at the BHL data harvest database, I can see that there was about a 50-day gap where no data was harvested from BioStor.  That gap corresponds to when BHL stopped getting BioStor author “id” values.  The authors harvested on May 20 had IDs, and the next authors harvested (on July 11) did not.  None of the authors harvested since have had IDs.

That time period appears to also correspond to when the address of the BioStor APIs changed from “direct.biostor.org” to simply “biostor.org”.  That change was applied to the BHL code on July 13.

So….. help?  It looks like something changed on your end.

MIKE
```

**Comment by rdmpage** [comment]
@mlichtenberg The author id from the old BioStor was not included in the CSL-JSON used to create the Elasticsearch export, which in turn populates the public front-end to BioStor. I was unaware o the role that this played in BHL's import (i.e., without an id a new BHL creator would be created).

**Comment by rdmpage** [comment]
Fixed with ea98826e6dc0db05917161f659ee38f9efaf94ce
